skglm/experimental/smooth_quantile_regressor.py

```diff
@@ -0,0 +1,553 @@
+import numpy as np
+from scipy import sparse
+from skglm.experimental.pdcd_ws import PDCD_WS
+from numba import jit
+from skglm import GeneralizedLinearEstimator
+from skglm.penalties import L1
+from skglm.experimental.solver_strategies import StageBasedSolverStrategy
+
+
+@jit(nopython=True)
```

Comment:
a priori there is no for loop, everything is vectorized so no need to JIT compile it